src/qibolab/compilers/default.py

```diff
@@ -27,11 +23,15 @@ def rz_rule(gate: Gate, qubit: Qubit) -> PulseSequence:
     return PulseSequence([(qubit.drive.name, VirtualZ(phase=gate.parameters[0]))])
 
 
-def gpi2_rule(gate: Gate, qubit: Qubit) -> PulseSequence:
+def identity_rule(gate: Gate, qubit: SingleQubitNatives) -> PulseSequence:
```

Comment:
True for all the rules: `qubit` name is probably not the best choice for a variable of type `SingleQubitNatives`/`TwoQubitNatives`/`list[SingleQubitNatives]`, but at least the type annotation makes it clearer.

Reply:
Yeah, I've been conservative, but I could have renamed it in `natives`.

Reply:
Done in 8b2c7d8d37b6803b9825d614b9437570bee90b4c